opds.py

```diff
@@ -1806,10 +1805,8 @@ def group_uri(self, work, license_pool, identifier):
         lanes = self.lanes_by_work.get(work, None)
 
         if lanes:
-            lane_name = lanes[0]['lane'].display_name
-            additional_lanes = lanes[1:]
-            if additional_lanes:
-                self.lanes_by_work[work] = additional_lanes
+            lane_dic = lanes.pop(0)
+            lane_name = lane_dic['lane'].display_name
```

Comment:
I don't remember what this change was, so I'm a little suspicious of it, but it's another bug that the Python 3 conversion caught. I remember it was causing the same work to show up multiple times in a list. And it doesn't seem like it belongs in a method that generates a URI.

Reply:
Looks like this only used for tests anyway.